includes/class-convertkit-setup.php

```diff
@@ -166,7 +166,7 @@ private function change_id_to_form_id() {
 		$mappings = $api->get_subscription_forms();
```

Comment:
I don't quite understand this one - why would `$api->get_subscription_forms()` return an error if there was no data - shouldn't it just be an empty array?

Reply:
It returns a `WP_Error` in tests due to hitting a rate limit, as we run tests in parallel.

Reply:
Would it help if we had infra increase the rate limit for the WordPress testing account? I don't want to introduce a catch-all exception handler in code like this - at the very least, we should be checking for a rate limit error explicitly, both in the plugin and tests.

Reply:
I think the rate limit is pretty high already, but if there's scope to increase it given we're running so many tests in parallel, sure.

Reply:
I have requested an increase.